src/ffi/pmplane.rs

```diff
@@ -1,46 +1,20 @@
-#![allow(unused, deref_nullptr)]
+// `pmplane_t` struct is from generated bindgen in playermove.rs
 
-use std::mem::{align_of, size_of};
+#![allow(unused, nonstandard_style, deref_nullptr)]
+
+use std::mem::{align_of, offset_of, size_of};
 use std::os::raw::*;
-use std::ptr::null;
 
 #[repr(C)]
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug)]
 pub struct pmplane_t {
```

Comment:
Is it the same struct, just with a C typedef or something? If so then can get rid of it

Reply:
Not sure what you mean by this. This is the same struct from the older bxt-rs bindgen. Albeit, pmplane.rs and pmtrace.rs aren't referenced anywhere outside of this `ffi` module.

Reply:
I mean, where does it come from the includes? Is it actually defined twice in two different C includes? Or one of the includes defines it, and some other include just typedefs it under a slightly different name (pmplane vs. plane)

Reply:
It seems like `pmplane_t` and `plane_t` do have exactly same data structure, but they are used differently. They are not type alias of one another. `trace_t` and `pmtrace_t` look similar but they have slightly different data structure. I think it is better just to keep things as is.